client/src/components/pages/Login.js

```diff
@@ -19,9 +20,18 @@ const Login = () => {
   const [email,setEmail]= useState("")
  const [isLoading,setIsLoading] = useState(false)
   const navigate=useNavigate()
+  const [loading,setLoading] = useState(false)
+  useEffect(()=>{
```

Comment:
Pls remove this useEffect and Loading. It is not good to make user wait if the data is already loaded just to show the loading animation.

Reply:
can i decrease the timing of set timeout because I want to show the loader